packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -1,25 +1,104 @@
 import * as React from 'react';
 
-import { Button, HeaderLayout } from '@strapi/design-system';
-import { CheckPermissions } from '@strapi/helper-plugin';
+import {
+  Box,
+  Button,
+  ContentLayout,
+  Flex,
+  HeaderLayout,
+  Main,
+  Tab,
+  TabGroup,
+  TabPanel,
+  TabPanels,
+  Tabs,
+} from '@strapi/design-system';
+import {
+  CheckPermissions,
+  LoadingIndicatorPage,
+  PageSizeURLQuery,
+  PaginationURLQuery,
+  useQueryParams,
+} from '@strapi/helper-plugin';
 import { Plus } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 
 import { AddReleaseDialog } from '../components/AddReleaseDialog';
+import { ReleasesGrid } from '../components/ReleasesGrid';
 import { PERMISSIONS } from '../constants';
+import { useGetReleasesQuery } from '../modules/releaseSlice';
+
+interface ReleasesPageQuery {
+  page?: number;
+  pageSize?: number;
+  filters?: {
+    $and?: Array<{
+      releasedAt?: {
+        $notNull?: boolean;
+      };
+    }>;
+  };
+}
```

Comment:
Same comment as above, and I think the type should be shared between the two pages 🙃